packages/react-dom/index.classic.fb.js

```diff
@@ -31,6 +31,7 @@ export {
   unstable_createEventHandle,
   unstable_renderSubtreeIntoContainer,
   unstable_runWithPriority, // DO NOT USE: Temporarily exposed to migrate off of Scheduler.runWithPriority.
+  useFormStatus as experimental_useFormStatus,
```

Comment:
Just noticing that the form flag is on in www but not the async transitions one.

Reply:
Yeah I intentionally kept the async one off because it affects error handling behavior even in regular, sync useTransition. Should we turn enableFormActions off too?

Reply:
Yea, let's do that for now.

Reply:
If we turn them on we have to also update this to get the right external runtime from experimental or make an fb build.

https://github.com/facebook/react/blob/main/.github/workflows/commit_artifacts.yml#L137-L138

Reply:
Ok I opened a separate PR where we can discuss: #26721